libvast/vast/type.hpp

```diff
@@ -825,6 +836,10 @@ class record_type final : public stateful_type_base {
   explicit record_type(std::initializer_list<field_view> fields) noexcept;
   explicit record_type(const std::vector<struct field>& fields) noexcept;
 
+  /// Returns the underlying FlatBuffers table representation.
+  /// @note Include `vast/fbs/type.hpp` to be able to use this function.
```

Comment:
I don't think we need to tell the user what to include in order to use a function.

Reply:
Fair point. I'll add a comment at the top of the `vast/type.hpp` file though to never include `vast/fbs/type.hpp` in the header, so reviewers don't miss this if IWYU added it by accident.